ranger/data/scope.sh

```diff
@@ -172,6 +172,8 @@ handle_image() {
 
         ## Video
         # video/*)
+        #     # check if file is incomplete torrent video file
+        #     [ "$FILE_EXTENSION_LOWER" = "part" ] && exit 1
```

Comment:
Maybe we should `break` instead? That way we still get the `file(1)` generated preview identifying the file.

Reply:
Hi, thanks for reviewing. 

Like this `[ "$FILE_EXTENSION_LOWER" = "part" ] && break`  ?

Reply:
I think that should work, yeah.

Reply:
But it is not working for me, still an image is getting previewed instead of going to [here](https://github.com/ranger/ranger/blob/1a007158cef1279df68d0e8567b126def6b68046/ranger/data/scope.sh#L452) .